src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ThemeManager3.cs

```diff
@@ -0,0 +1,434 @@
+using Standard;
+using Microsoft.Win32;
+using System.Collections;
+using System.Collections.Generic;
+using System.Windows.Interop;
+using System.Windows.Media;
+using System.Windows.Appearance;
+
+namespace System.Windows;
+
+public static class ThemeManager3
+{
+    static ThemeManager3()
+    {
+        if(Application.Current != null)
```

Comment:
could do Application.Current?.Theme == "None"

(btw this whole file seems to have weird code style with no space before opening parenthesis

Reply:
Yeah just updated that. Btw take a look at ThemeManager3 only. I am removing ThemeManager and replacing it with ThemeManager3 ( I will rename this in main PR )